addons/nl_ums/models/student.py

```diff
@@ -0,0 +1,35 @@
+from odoo import models,fields,api
```

Comment:
I see this file has lots of code style issues. Please run autopep8 as I told you guys yesterday.

@danish-netlinks @ibrahim-netlinks @Nazary-Netlinks @mitra-netlinks please read about [PEP 8](https://www.python.org/dev/peps/pep-0008/) here and fix your code accordingly. There are editor and IDE extensions that shows theses errors to you.